<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createEmptyDBDialogBase</class>
 <widget class="QDialog" name="createEmptyDBDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>444</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Daten einfügen</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Datenimport starten ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string comment="Pfade der Import- und Exportdatenbank"/>
   </property>
   <property name="title">
    <string>QKan-Datenbank</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>QKan-Datenbank (*.sqlite):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_selectQKanDB">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zieldatenbank auswählen und optional erstellen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tf_qkanDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QKan - Zieldatenbank (wird ggfs. neu angelegt)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string comment="Pfade der Import- und Exportdatenbank"/>
   </property>
   <property name="title">
    <string>Spaltennamen kopieren</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lbSchaechte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Schächte:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auslässe:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speicher:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Haltungen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pumpen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wehre:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopyWehre">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wehrdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Wehrdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopySpeicher">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Speicherdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Speicherdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopyHaltungen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Haltungsdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Haltungsdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopyPumpen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pumpendaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Pumpendaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopyAuslaesse">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Auslassdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Auslassdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCopySchaechte">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Schachtdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Schachtdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string comment="Pfade der Import- und Exportdatenbank"/>
   </property>
   <property name="title">
    <string>Tabellendaten aus Clipboard einfügen</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lbSchaechte_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Schächte:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auslässe:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speicher:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Haltungen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pumpen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbSchaechte_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wehre:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPasteSchaechte">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Schachtdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Schachtdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPasteWehre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wehrdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Wehrdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPasteSpeicher">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Speicherdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Speicherdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPasteHaltungen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Haltungsdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Haltungsdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPastePumpen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pumpendaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Pumpendaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPasteAuslaesse">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Auslassdaten aus Clipboard übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Auslassdaten aus Clipboard übernehmen</string>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>createEmptyDBDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>createEmptyDBDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
